src/scss/style.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  font-family: Roboto, sans-serif;
+}
```

Comment:
```suggestion

```
Consider moving general styles to a separate file

Reply:
I'm sorry, I didn't quite understand what needs to be fixed in src/scss/elements/_product-card.scss in line 15 and 21-25